pygridtools/iotools.py

```diff
@@ -233,3 +234,165 @@ def interactive_grid_shape(grid, max_n=200, plotfxn=None, **kwargs):
         plotfxn=ipywidgets.fixed(plotfxn),
         plotopts=ipywidgets.fixed(kwargs)
     )
+
```

Comment:
PEP8 wants two lines here. See the CI log or run `python check_pygridtools --strict` locally to see all PEP8 errors

Reply:
I've been meaning to try ambv/black - mind if I give this a go here?

Reply:
that'll dramatically change everything in the file. if can run want to run it on this bit pertinent to this PR in isolation, go for it.

running `black` over the whole code base in separate PR would be welcome.